src/serializableStateInvariantMiddleware.test.ts

```diff
@@ -112,18 +112,14 @@ describe('serializableStateInvariantMiddleware', () => {
     expect(action).toBe(dispatchedAction)
   })
 
-  it('Should log an error when a non-serializable value is in state', () => {
-    const ACTION_TYPE = 'TEST_ACTION'
-
+  function dispatchActionWithValueInTestStore(middleware: Middleware, actionType: string, badValue: any) {
```

Comment:
Not sure you need to introduce this.  You can get by setting up the store in each test (like all the other tests).

Reply:
OK — I got rid of it and updated the PR.